docs/conf.py

```diff
@@ -78,13 +60,6 @@ def get_version(release):
 project = 'aiomysql'
 copyright = '2015,2016 Nikolay Novik'
 
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-release = get_release()
-version = get_version(release)
```

Comment:
These are Sphinx's built-in settings, with these exact names. They should be kept. I suggest adding `setuptools-scm` as a docs build dependency. `get_version()` could be imported and generate both of these vars. The only extra thing to take care of is making sure to `git fetch --tags --all` in the local docs building wrapper. I like this integration in the docs because then I can show the precise commit sha in the build.

Reply:
How about doing the Sphinx + setuptools-scm integration into a completely separate effort?

Reply:
moved to #748